Ical.Net/Evaluation/EvaluationPeriod.cs

```diff
@@ -0,0 +1,52 @@
+﻿//
+// Copyright ical.net project maintainers and contributors.
+// Licensed under the MIT license.
+//
+
+using System;
+using NodaTime;
+
+namespace Ical.Net.Evaluation;
+public class EvaluationPeriod : IComparable<EvaluationPeriod>
```

Comment:
Could all Evaluation classes become internal. Would increase flexibility for future changes without causing a breaking change?